modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -6,55 +6,50 @@
 [id="nw-ingress-converting-http-header-case_{context}"]
 = Converting HTTP header case
 
-HAProxy lowercases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+HAProxy lowercases HTTP header names by default; for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
 
 [IMPORTANT]
 ====
-Because {product-title} includes HAProxy 2.8, be sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading.
+{product-title} includes HAProxy 2.8. If you want to upgrade to this version of the web-based load balancer, ensure that add `spec.httpHeaders.headerNameCaseAdjustments` to your cluster's configuration file.
 ====
 
+As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by setting the `HeaderNameCaseAdjustments` field in the Ingress Controller YAML file.
+
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
 
-As a cluster administrator, you can convert the HTTP header case by entering the `oc patch` command or by setting the `HeaderNameCaseAdjustments` field in the Ingress Controller YAML file.
+* Capitalize an HTTP header by using the `oc patch` command.
 
-* Specify an HTTP header to be capitalized by entering the `oc patch` command.
-
-. Enter the `oc patch` command to change the HTTP `host` header to `Host`:
+. Enter the `oc patch` command to change the HTTP header from `host` to `Host`:
 +
 [source,terminal]
 ----
 $ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"httpHeaders":{"headerNameCaseAdjustments":["Host"]}}}'
 ----
 +
-. Annotate the route of the application:
-+
-[source,terminal]
-----
-$ oc annotate routes/my-application haproxy.router.openshift.io/h1-adjust-case=true
-----
-+
-The Ingress Controller then adjusts the `host` request header as specified.
-
-//Extra example if needed
-////
-* This example changes the HTTP `cache-control` header to `Cache-Control`:
+. Create a `Route` resource YAML file so that the annotation can be applied to the application.
```

Comment:
Similar comment about making this a level 2 list item:
```suggestion
.. Create a `Route` resource YAML file so that the annotation can be applied to the application.
```

Reply:
:heavy_check_mark: